ccm_web/server/logger.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path'
+import winston, { format } from 'winston'
+
+const logFormat = format.printf(
+  ({ level, message, timestamp, filePath }) => {
+    const moduleName = path.basename(filePath)
+    return `${String(timestamp)} - ${moduleName} - ${level} - ${message}`
+  }
+)
+
+const baseLogger = winston.createLogger({
+  transports: [new winston.transports.Console()],
+  format: format.combine(
+    format.timestamp(),
+    format.colorize(),
```

Comment:
This might be something we want to be able to disable. It probably looks good in dev but not sure what this is going to look like on Openshift or Kibana?

Reply:
Hm, was trying to keep configuration out of this file, but we could branch on this based on `NODE_ENV` (`production` or `development`), or just remove it. What do you think? We could also wait until this is deployed in a QA setting and see what happens.

Reply:
I'm going to wait before changing this, but I'm glad you flagged in, I am not sure how OpenShift will respond.